app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -16,7 +16,9 @@ def document_capture_session_poll_render_result
       document_capture_session = DocumentCaptureSession.find_by(uuid: session_uuid)
       return { plain: 'Unauthorized', status: :unauthorized } unless document_capture_session
 
-      result = document_capture_session.load_result
+      result = document_capture_session.load_result ||
+               document_capture_session.load_doc_auth_async_result
```

Comment:
reason this is needed: the VerifyDocumentAction stores via `store_doc_auth_result` which still contains PII but has slightly different redis keys than the `store_result_from_response` that the old flow uses